.travis.yml

```diff
@@ -1,3 +1 @@
 language: node_js
-node_js:
-  - "4.4"
```

Comment:
I imagine we will still want to specify a version of node here, maybe match that in the .nvmrc file.

Reply:
[Travis falls back to what’s in .nvmrc](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions-using-.nvmrc). Does anything other than Travis use this file?

Reply:
Ok, that works then.